src/main/java/com/vmware/vipclient/i18n/base/instances/TranslationMessage.java

```diff
@@ -147,6 +148,65 @@ private String getMessageWithArgs(String resourceBundle, final Locale locale, fi
         return FormatUtils.formatMsg(message, Locale.forLanguageTag(msgsItemDTO.getLocale()), args);
     }
 
+    /**
+     * Get one version's key based message if there is multiple versions for one product, the message's arguments are put in a variable object
+     *
+     * @param locale
+     * @param version
+     * @param component
+     * @param key
+     * @param args
+     * @return
+     */
+    public String getMultiVersionMessage(final Locale locale, final String version, final String component, final String key, final Object... args) {
```

Comment:
Since the requirement of product is to support multiple versions, so design the version in config file can be multiple versions, such as 'all' or '1.0.0,2.0.0', but when the string appear on UI, it must be of one version, so need specify it in the method parameter.